Vagrantfile

```diff
@@ -11,60 +11,76 @@ VM_GUI=false
 GOROOT = '/opt/go'
 GOPATH = '/opt/gopath'
 PACKAGE_PATH = 'src/github.com/watson-platform/packer-builder-softlayer'
+GO_VERSION = '1.9.2'
 
 script = <<SCRIPT
+set -x
+
 SRCROOT="#{GOROOT}"
 
 # Install Go
-sudo apt-get update
-sudo apt-get install -y build-essential mercurial
-sudo hg clone -u release https://code.google.com/p/go $SRCROOT
-cd ${SRCROOT}/src
-sudo ./all.bash
-
-# Setup the GOPATH
-sudo mkdir -p #{GOPATH}
-cat <<EOF >/tmp/gopath.sh
+
+download="https://storage.googleapis.com/golang/go#{GO_VERSION}.linux-amd64.tar.gz"
+
+wget -q -O /tmp/go.tar.gz ${download}
+
+tar -C /tmp -xf /tmp/go.tar.gz
+sudo mv /tmp/go /usr/local
+sudo chown -R root:root /usr/local/go
+
+# Ensure that the GOPATH tree is owned by vagrant:vagrant
+mkdir -p #{GOPATH}
+chown -R vagrant:vagrant #{GOPATH}
+
+# Ensure Go is on PATH
+if [ ! -e /usr/bin/go ] ; then
+  ln -s /usr/local/go/bin/go /usr/bin/go
+fi
+if [ ! -e /usr/bin/gofmt ] ; then
+  ln -s /usr/local/go/bin/gofmt /usr/bin/gofmt
+fi
+
+
+# Ensure new sessions know about GOPATH
+if [ ! -f /etc/profile.d/gopath.sh ] ; then
+  cat <<EOT > /etc/profile.d/gopath.sh
 export GOPATH="#{GOPATH}"
-export PATH="#{GOROOT}/bin:\\$GOPATH/bin:\\$PATH"
-export SL_USERNAME=#{ENV['SL_USERNAME']}
-export SL_API_KEY=#{ENV['SL_API_KEY']}
-EOF
-sudo mv /tmp/gopath.sh /etc/profile.d/gopath.sh
-sudo chmod 0755 /etc/profile.d/gopath.sh
+export PATH="#{GOPATH}/bin:\$PATH"
+EOT
```

Comment:
EOT?

Reply:
Just a heredoc for adding that text to `gopath.sh`